src/ControlSystem/ControlErrors/Size.hpp

```diff
@@ -230,35 +301,100 @@ struct Size : tt::ConformsTo<protocols::ControlError> {
             QueueTags::InverseSpatialMetricOnExcisionSurface<Frame::Distorted>>(
             excision_quantities);
 
```

Comment:
(not a change request) why is `operator()` defined here in the hpp instead of implemented in the cpp file?

Reply:
The template parameters of `operator()` are not known.